tests/test_multithreaded.rs

```diff
@@ -13,18 +13,16 @@
 // limitations under the License.
 //
 extern crate rocksdb;
-mod util;
 
-use rocksdb::DB;
+use rocksdb::{TemporaryDBPath, DB};
 use std::sync::Arc;
 use std::thread;
-use util::DBPath;
 
 const N: usize = 100_000;
 
 #[test]
 pub fn test_multithreaded() {
-    let n = DBPath::new("_rust_rocksdb_multithreadtest");
+    let n = TemporaryDBPath::new("_rust_rocksdb_multithreadtest");
```

Comment:
@iSynaptic What do you think about to generate random name inside `new` method and change api to `TemporaryDBPath::new()` ?

Reply:
I like it. I'll make the change.

Reply:
Done. I needed to add a little logic to `TemporaryDBPath` to disambiguate directories when running multi-threaded, but I think it get it to where we wanted.